wtf fails to start with resourceusage widget enabled in Fedora 30

## What's broken?
Since a couple of days the resourceusage widget crashes wtfutil on boot with the following message:

```
panic: runtime error: index out of range [4] with length 3 [recovered]
	panic: runtime error: index out of range [4] with length 3

goroutine 1 [running]:
github.com/rivo/tview.(*Application).Run.func1(0xc0006bf880)
	/Users/ccummer/go/pkg/mod/github.com/rivo/tview@v0.0.0-20190829161255-f8bc69b90341/application.go:149 +0x82
panic(0x1ce3480, 0xc000b32dc0)
	/usr/local/Cellar/go/1.13.1/libexec/src/runtime/panic.go:679 +0x1b2
github.com/wtfutil/wtf/modules/resourceusage.MakeGraph(0xc0001cadc0)
	/Users/ccummer/go/src/github.com/wtfutil/wtf/modules/resourceusage/widget.go:108 +0x9ef
github.com/wtfutil/wtf/modules/resourceusage.display(...)
	/Users/ccummer/go/src/github.com/wtfutil/wtf/modules/resourceusage/widget.go:138
github.com/wtfutil/wtf/modules/resourceusage.(*Widget).Refresh.func1()
	/Users/ccummer/go/src/github.com/wtfutil/wtf/modules/resourceusage/widget.go:131 +0x78
github.com/rivo/tview.(*Application).QueueUpdateDraw.func1()
	/Users/ccummer/go/pkg/mod/github.com/rivo/tview@v0.0.0-20190829161255-f8bc69b90341/application.go:493 +0x2f
github.com/rivo/tview.(*Application).Run(0xc0006bf880, 0x0, 0x0)
	/Users/ccummer/go/pkg/mod/github.com/rivo/tview@v0.0.0-20190829161255-f8bc69b90341/application.go:253 +0x615
main.main()
	/Users/ccummer/go/src/github.com/wtfutil/wtf/main.go:73 +0x37e
```
I am using Fedora 30 with the config:

```
resources:
      type: resourceusage
      enabled: true
      position:
        top: 1
        left: 0
        height: 2
        width: 1
      refreshInterval: 1
```

--- a/modules/resourceusage/widget.go
+++ b/modules/resourceusage/widget.go
@@ -105,7 +105,7 @@ func MakeGraph(widget *Widget) {
 	usedSwapLabel := bytefmt.ByteSize(swapUsed)
 	totalSwapLabel := bytefmt.ByteSize(memInfo.SwapTotal)
 
-	if usedSwapLabel[len(usedSwapLabel)-1] == totalMemLabel[len(totalSwapLabel)-1] {
+	if usedSwapLabel[len(usedSwapLabel)-1] == totalSwapLabel[len(totalSwapLabel)-1] {
 		usedSwapLabel = usedSwapLabel[:len(usedSwapLabel)-1]
 	}
 

